release-stuff/build-macos.sh

```diff
@@ -41,6 +41,7 @@ cd $DIR/..
 
 # Download precompiled perl.
 mkdir download
+unset PERL5LIB
```

Comment:
All of these `release-stuff/build-*` scripts are used in an automated CI toolchain. So the environment is well defined (and it did work when doing the last release).

But still, is it good style to always undefine `PERL5LIB` in scripts that set up their own perl environment? If yes, can you do the respective change in the other build scripts as well? When doing the next release it will become obvious if the changes break the release process or not.

Reply:
The point of unsetting the var is to get `pp` working. Because otherwise cpanm installs things into `~/perl5`, in my case and then `pp` fails to locate `Time::Local`. I didn't know the script is for CI use only because I needed to build a working `rakubrew` for tests and the thing worked for me after the fix. If there is another way to produce testable binary I'd be ok with it. :)

In either case, I see no problem in resetting `PERL5LIB` because the build happens in a localized environment where everything should be controllable by the build. `PERL5LIB` introduces an external uncontrolled environment into the build. I will add it into the other build scripts.

Reply:
Looked into other build scripts. `build-linux.sh` doesn't need unsetting of `PERL5LIB` because it's using the default perl. `build-windows.ps1` is likely to need it too because it is also downloading and installing local copy of strawberry perl. But I'm the lucky one to never use PowerShell before and don't know how to unset a variable.

Reply:
I'll do it.